web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala

```diff
@@ -96,9 +96,23 @@ import scala.util.{Failure, Success}
     )
 
     val error = formData.firstValidationError.map { text =>
+      val emailNotVerified = "The email is not verified, check your spam folder if you don't see the email."
+
       Container(
+        alignItems = Alignment.center,
         margin = Container.EdgeInsets.top(16),
-        child = ErrorLabel(text)
+        child = Fragment(
+          ErrorLabel(text),
+          text match {
```

Comment:
And personally I'd move this pattern matching to a new variable `resendVerifyEmailButton`, that way we make it easier to read and modularize

Reply:
I'd actually create a helper function which picks the error text from code that's shared between frontend/backend